src/Methods.cs

```diff
@@ -91,61 +96,66 @@ public async Task<PackageList> GetNugetInformationAsync(string project, IEnumera
                             continue;
                         }
 
-                        // Search nuspec in local cache
+                        // Search nuspec in local cache (Fix for linux distro)
                         string userDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
-                        var nuspecPath = Path.Combine(userDir, ".nuget", "packages", packageWithVersion.Name, version, packageWithVersion.Name + ".nuspec");
+
+                        //Linux: package file name is lowercase
+                        var packageNameTmp = IsLinux() ? packageWithVersion.Name?.ToLowerInvariant() : packageWithVersion.Name;
```

Comment:
I am not sure if there is any difference between linux and windows for nuget. I've checked both Ubuntu and Windows packages folder .nuget\packages and both package names are the same. Is there any example ?